src/components/dataset/overview/connection-to-forms.vue

```diff
@@ -39,44 +41,35 @@ except according to the terms contained in the LICENSE file.
 
 <script>
 import ExpandableRow from '../../expandable-row.vue';
+import FormLink from '../../form/link.vue';
 import I18nList from '../../i18n/list.vue';
 import SummaryItem from '../../summary-item.vue';
 
 import useRoutes from '../../../composables/routes';
+import { useRequestData } from '../../../request-data';
 
 export default {
   name: 'ConnectionToForms',
   components: {
     ExpandableRow,
+    FormLink,
     I18nList,
     SummaryItem
   },
-  props: {
-    properties: {
-      type: Array,
-      required: true
-    },
-    sourceForms: {
-      type: Array,
-      required: true
-    },
-    projectId: {
```

Comment:
Now that `projectId` is added to each form, we don't need to pass it in as a prop. I went ahead and removed that prop, but then there was an issue in the tests of the component. The component now expects the forms to have a `projectId` property, but in the tests of the component, they didn't. The easiest way to solve that was to specify a dataset for `requestData`. Doing so automatically transforms the dataset object, adding `projectId`. Once I was passing the dataset to the test, I felt like it didn't make sense to also pass `properties` and `sourceForms` as props, since those are accessible from the dataset.